### ISSUE TITLE
Should we offer a prompt to upgrade package version (at least for whitelisted/known libraries)?

### ISSUE THREAD

**Issue by Zlatkovsky** [OfficeDev/script-lab#434]
Concrete example:  we've now bumped Office JS version twice, 0.7.1 for OfficeHelpers.UI.notify, and 0.7.2 for a fix to Office Online sheet activation.

When users already have an existing snippet that uses an older version, should we prompt them to update?...

**Comment by jakobpn** [comment]
Yes, good idea. I would recommend doing this as a modeless notification that can be dismissed. Can we also check for never version of npm packages?

**Comment by Zlatkovsky** [comment]
Marking as "high-impact" since week running into this (in the sense of updating OfficeHelpers often)

**Comment by WrathOfZombies** [comment]
Question: how would you get the versions of the libraries and how would you prompt people? Two is super annoying when you end up prompting me when I dont want it. 

Case in point, the VSCode Icons extension always tells me about an update/modify my settings.json file which is annoying.

**Comment by WrathOfZombies** [comment]
We should keep our samples and starter snippets updated but leave folks have a choice on the package they want to use. My intellisense helper for libraries can include the latest version number instead to show that's the latest version as opposed to what you have.

**Comment by WrathOfZombies** [comment]
https://github.com/OfficeDev/script-lab/pull/440

**Comment by WrathOfZombies** [comment]
Another example of annoying prompts everytime I open vscode. 

![image](https://cloud.githubusercontent.com/assets/3244360/26741442/397cbcc4-478e-11e7-8b53-0529497eb792.png)

**Comment by Zlatkovsky** [comment]
Agreed that a prompt is annoying.

What if we did a silent update, but only for OfficeHelpers (since that's a pckage we have full control over, and we've now updated it ~5 times)?  @WrathOfZombies, @Jakobpn, what do you think?

**Comment by WrathOfZombies** [comment]
umm... Not sure if that makes me feel any better. Please dont. Let the developer decide what to do with their snippets. You can do the upgrade on the new template and the samples. Please reserve silent upgrades to things like upgrading to a new snippet style or a new version of Script Lab etc.
